unittests/contracts/jumborow/CMakeLists.txt

```diff
@@ -1,2 +1,9 @@
-configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/jumborow.wasm ${CMAKE_CURRENT_BINARY_DIR}/jumborow.wasm COPYONLY )
+if(EOSIO_COMPILE_TEST_CONTRACTS)
+   add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/jumborow.wasm"
+                      COMMAND "${CDT_ROOT}/bin/eosio-wast2wasm" "${CMAKE_CURRENT_SOURCE_DIR}/jumborow.wast" -o "${CMAKE_CURRENT_BINARY_DIR}/jumborow.wasm"
+                      DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/jumborow.wast")
+   add_custom_target(gen_jumborow_wasm ALL DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/jumborow.wasm")
```

Comment:
Using `ALL` is an anti-pattern imo. But since we're building this with ExternalProject, and then
https://github.com/AntelopeIO/spring/blob/f871bf2c6356cc6273473aa2ab0ac8ddcf7f4f21/unittests/CMakeLists.txt#L43
on top of that, doesn't really seem to be too offensive to stuff this in `ALL`